events.go

```diff
@@ -137,6 +137,22 @@ type Event interface {
 // concrete event types.
 type BasicEvent struct{ *js.Object }
 
+type EventInit struct {
```

Comment:
Perhaps it'd be better (more idiomatic) to name this struct `EventOptions`? Similar to [`PositionOptions`](https://github.com/arvitaly/go-js-dom/blob/9af2ce4cd4e5d5782e90b05dca66f73c88f97582/dom.go#L1242) or [`imports.Options`](https://godoc.org/golang.org/x/tools/imports#Options).